config/config.go

```diff
@@ -55,6 +58,51 @@ type Config struct {
 	Auth     Auth     `yaml:"auth" json:"auth"`
 	Trace    Trace    `yaml:"trace" json:"trace"`
 	Metrics  Metrics  `yaml:"metrics" json:"metrics"`
+	Updaters Updaters `yaml:"updaters" json:"updaters"`
+}
+
+// Updaters configures updater behavior.
+type Updaters struct {
```

Comment:
do we need to update the configs used in our repos now?

There is a sample config at the root of the repo,
./local-dev/clair/config.yaml is used in local dev.